bridge/engine/ActiveMediaList.cpp

```diff
@@ -355,12 +384,35 @@ void ActiveMediaList::process(const uint64_t timestampMs, bool& outDominantSpeak
 
     auto dominantSpeaker = heap.top();
 
+    if (!_c9_conference)
+    {
+        _activeTalkers.clear();
+    }
+
     for (size_t i = 0; i < _audioLastN && !heap.empty(); ++i)
     {
-        updateActiveAudioList(heap.top().participant);
+        auto top = heap.top();
+        updateActiveAudioList(top.participant);
+        if (!_c9_conference)
+        {
+            if (top.score > top.noisePlus20percent)
```

Comment:
I would rather calculate the noise floor * 1.2 here to avoid name that includes "calculation". Just call it noiseLevel and let rankSpeakers do only what the name implies, rank Speakers. If you change the margin later, we do not want to change the name.

Reply:
A question regarding the margin. We have a score based on diff in dB basically. Is it the right thing to *1.2 or is a +12 more real?

Reply:
Fixed to configurable dbDiff value (default is tested 18 dB)